TombEngine/Scripting/Internal/TEN/Strings/DisplayString/DisplayString.cpp

```diff
@@ -181,6 +186,16 @@ std::string DisplayString::GetKey() const
 	return s.m_key;
 }
 
+void DisplayString::SetFlags(sol::table const& flags) {
+	UserDisplayString& s = s_getItemCallback(m_id).value();
```

Comment:
Please aim to use variable names longer than a single character as they get pretty vague.

Reply:
I used the same convention as the other functions. They use a variable **s** to set a parameter or return a value. However, I can modify the function I created to be clearer